api/src/main/java/com/velocitypowered/api/proxy/player/TabList.java

```diff
@@ -45,11 +44,18 @@ public interface TabList {
   void addEntry(TabListEntry entry);
 
   /**
-   * Adds a {@link List} of {@link TabListEntry}'s to the {@link Player}'s tab list.
+   * Adds a {@link Collection} of {@link TabListEntry}'s to the {@link Player}'s tab list.
    *
    * @param entries to add to the tab list
    */
-  void addEntries(List<TabListEntry> entries);
+  void addEntries(Collection<TabListEntry> entries);
+
+  /**
+   * Adds an array of {@link TabListEntry}'s to the {@link Player}'s tab list.
```

Comment:
Don't really like the word "array" here but couldn't come up with something better